hw/ip/rv_dm/dv/rv_dm_sim_cfg.hjson

```diff
@@ -73,6 +76,14 @@
 
   // List of test specifications.
   tests: [
+    {
+      name: "rv_dm_csr_aliasing"
+      build_mode: "cover_reg_top"
+      run_opts: ["+csr_aliasing"]
+      en_run_modes: ["csr_tests_mode"]
+      reseed: 5
```

Comment:
Why do you decrease the reseed by so much? In the life cycle controller they reseed 10 times.

Reply:
This is the default from `csr_tests`. I have not changed it.